pkg/autoscaler/config.go

```diff
@@ -37,8 +37,8 @@ type Config struct {
 	EnableScaleToZero bool
 
 	// Target concurrency knobs for different container concurrency configurations.
-	ContainerConcurrencyTargetPercentage float64
-	ContainerConcurrencyTargetDefault    float64
+	ContainerConcurrencyTargetFraction float64
```

Comment:
nit: Ratio?  Fractions have numerators and denominators :)

Reply:
As discussed in person :-)